src/main/java/org/jenkinsci/plugins/workflow/cps/UninstantiatedDescribableWithInterpolation.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable;
+
+import java.util.Map;
+import java.util.Set;
+
+public class UninstantiatedDescribableWithInterpolation extends UninstantiatedDescribable {
+    private final Set<String> interpolatedStrings;
```

Comment:
```suggestion
    private static final long serialVersionUID = 1L;
    private final Set<String> interpolatedStrings;
```

Reply:
added in b828472cff3abc7ccce59127b6a8a1e5845a2eb3